src/main/scala/tech/cryptonomic/conseil/Conseil.scala

```diff
@@ -53,6 +53,11 @@ object Conseil extends App with LazyLogging with EnableCORSDirectives with Conse
         case Success(_) => logger.info("Pre-caching successful!")
       }
 
+      tezosPlatformDiscoveryOperations.initAttributesCache.onComplete {
```

Comment:
Did we consider doing retries? Maybe in a separate PR?

Reply:
I think it should be don in other PR for this and for preCaching